src/main/java/io/github/syst3ms/skriptparser/log/SkriptLogger.java

```diff
@@ -218,7 +237,7 @@ public void clearLogs() {
      * Finishes a logging process by making some logged entries definitive. All non-error logs are made definitive
      * and only the error that has the most priority is made definitive.
      */
-    public void logOutput() {
+    public void finalizeLogs() {
```

Comment:
That's funny, I also considered renaming this method, and thought about naming it commitLogs(). But your name makes more sense I think.